test/OrchardCore.Tests/Localization/PortableObjectStringLocalizerTests.cs

```diff
@@ -200,6 +200,26 @@ public void LocalizerReturnsOriginalTextForPluralIfTranslationDoesntExist(string
             Assert.Equal(expected, translation);
         }
 
+        [Theory]
+        [InlineData("zh-Hans", "zh", "球", 1, new string[] { "球" })]
+        [InlineData("zh-Hans", "zh", "球", 2, new string[] { "球" })]
+        public void LocalizerReturnsCorrectTranslationForPluralIfNoPluralFormsSpecified(string culture, string parentCulture, string expected, int count, string[] translations)
+        {
+            var translationCulture = CultureInfo.GetCultureInfo(parentCulture);
+            var currentCulture = CultureInfo.GetCultureInfo(culture);
+            CultureInfo.CurrentUICulture = currentCulture;
+
+            // using DefaultPluralRuleProvider to test it returns correct rule
+            TryGetRuleFromDefaultPluralRuleProvider(currentCulture, out var rule);
+            Assert.NotNull(rule);
+
+            SetupDictionary(culture, new[] { new CultureDictionaryRecord("ball", null, translations), }, rule);
```

Comment:
Why the pluralization doesn't setup above like en, cz and ar?

Reply:
Using `DefaultPluralRuleProvider` to make sure my change work.  Otherwise I think it is unrelated to the change and it is redundant.